scrabbleScoreExercise/scrabbleScore.js

```diff
@@ -0,0 +1,69 @@
+// Programme: takes each letter of string input and saves it in an array
+//loops through each letter, saves corresponding score for the letter
+// scores are then summed up and scrabble score is displayed back to user
+'use strict'
```

Comment:
@NosiphoManyase  What does this line do?

Reply:
I saw it in one of the fundamentals videos. I don't remember fully, but apparently it reduces chances of getting bugs. The one time i forgot a " let i = 0" in a for-loop, it wouldn't execute the code. but it would execute the code when I commented that line out.